EU4toV2/Source/Mappers/DeadDefinitionMapper/DeadDefinitionMapping.h

```diff
@@ -0,0 +1,33 @@
+#ifndef DEAD_DEFINITION_MAPPING_H
+#define DEAD_DEFINITION_MAPPING_H
+#include "Parser.h"
+#include <string>
+
+namespace mappers
+{
+struct DeadDefinition
+{
+	std::string tag;
+	std::optional<std::string> culture;
+	std::optional<std::string> religion;
+	std::optional<std::string> government;
+	std::optional<bool> civilized;
```

Comment:
An optional around the bool seems unnecessarily complex. Just set a default for the bool and make it match the default for V2Country

Reply:
In regular circumstances, I'd agree, but not in this case. This is an override, and I need to know why I'm overriding civilized. Dead nations aren't only the ones we load from disk, but also those that lost their provinces in transition and those that got merged, but otherwise coming from EU4. 

These nations have civilized status calculated same as any country with three states (yes/no/unset for EU4/EU4/disk), and having a default here of true of false will lead us to a matrix of override/regularCalc 3x3 large, with 9 resulting cases. This approach sidesteps all that uncertainty.